datagateway_api/src/datagateway_api/icat/filters.py

```diff
@@ -88,6 +91,28 @@ def create_filter(self):
                 self.value = "(NULL)"
 
             where_filter = self.create_condition(self.field, "in", self.value)
+        elif self.operation == "inq":
+            self.operation = "in"
+            where_filter = self.create_filter()
```

Comment:
If we decide the above suggestion is the better way to do it, please make the same change to this operator too.